frontend/src/components/core/AppView/styled-components.ts

```diff
@@ -101,6 +100,13 @@ export const StyledAppViewBlockContainer =
     }
   )
 
+export const StyledAppViewBlockSpacer = styled.div(({ theme }) => {
+  return {
+    width: theme.sizes.full,
+    flexGrow: 1,
+  }
+})
```

Comment:
It looks like we don't really add docstrings for these little styled visual elements, but could we start? Like: what does this element do, and where/why do we use it?

Reply:
I'm happy to though convention/nomenclature explains this. Let me know if you feel it's worth explicitly saying.

- Spacer is a common term for just an element for the sole purpose to take up space.
- convention shows that the use of this is to provide a spacer in the AppView. Perhaps it makes sense for the comment to be in there to see why this Spacer is included here?

Reply:
I assumed this was an AppView-specific element - a "block spacer that we only use in AppView". But if it's just a generic spacer that we'd use in a bunch of places, should it instead live in the top-level `styled-components` (as just `StyledBlockSpacer`)?

Reply:
I think there's a reason to look into writing up some generic styled components, but this might not be the best example to put there at the moment. In a way, these are spacers that fit in the flexbox framework where other spacers might require different layout engines, which adds more complexity for this case.